### PR TITLE
disable portable on MacOS

### PR DESCRIPTION
**Pull request by zhangjinpeng1987**
Disable portable on MacOS to sidestep the compiler bug in clang 4.9.

### CODE REVIEW

**Comment by huachaohuang** [comment]
Any harm doing this?

**Comment by zhangjinpeng1987** [comment]
`Any harm doing this?`
@huachaohuang  No.

**Comment by huachaohuang** [comment]
LGTM

File: Makefile [hunk 1]
```diff
@@ -1,5 +1,10 @@
 ENABLE_FEATURES ?= default
 
+# Disable portable on MacOS
```
Comment (siddontang) [inline Makefile]: can you explain why should we need to do?

**Comment by BusyJay** [comment]
I thought you need to enable portable when compiling on macOS.

**Comment by zhangjinpeng1987** [comment]
@BusyJay `I thought you need to enable portable when compiling on macOS.`
What is the reason?

File: Makefile (hunk 1 above)
Comment (siddontang) [reply Makefile]: any update?

**Comment by siddontang** [comment]
Rest LGTM

**Pull request merged**